creditor_bank_account_service.go

```diff
@@ -152,31 +156,36 @@ func (s *CreditorBankAccountService) Create(ctx context.Context, p CreditorBankA
 	return result.CreditorBankAccount, nil
 }
 
+type CreditorBankAccountListParamsCreatedAt struct {
+	Gt  string `url:"gt,omitempty" json:"gt,omitempty"`
```

Comment:
@vdandugc just wondering if we want this to be `GreaterThan` instead of `Gt`, I am okay with `Gt` if this what other client libs are also using.

Reply:
Please ignore this then